publish-pr-preview/README.md

```diff
@@ -38,7 +51,8 @@ jobs:
       with:
         NPM_PUBLISH: npm run my-script
         IGNORE: folder/example_package folder/example_package2
+        SCOPES: minkimcello@npm thefrontisde@gpr
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
         NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
Do you need this? ( I don't think so )

Reply:
The README explains it pretty clearly. That example workflow config won't confuse anybody (unless they jump down to the example but then that's their fault).